src/app/mail/mail-settings/addresses-signature/addresses-signature.component.html

```diff
@@ -495,7 +495,7 @@ <h5 class="ui-header-subtitle text-dark mb-0">
             {{ mailbox.fingerprint }}
           </div>
           <div class="col-2 mailbox-list-row-item">
-            {{ mailbox.key_type === 'RSA4096' || !mailbox.key_type ? 'RSA 4096 BIT' : 'ECC' }}
+            {{ mailbox.key_type === 'RSA4096' || !mailbox.key_type ? 'RSA4096 BIT' : 'ECC' }}
```

Comment:
The idea was to remove the ` BIT` part. To be consistent with alll the other places.